### ISSUE TITLE
LinkedIn OAuth2 broken

### ISSUE THREAD

**Issue by makslevental** [python-social-auth/social-core#178]
repeatedly get

`requests.exceptions.HTTPError: 500 Server Error: Internal Server Error for url: https://api.linkedin.com/v1/people/~:(email-address,first-name,id,last-name)?format=json&oauth2_access_token=<snip>`

problem is here:

https://github.com/python-social-auth/social-core/blob/master/social_core/backends/linkedin.py#L88

it should be

```python
        return self.get_json(
            self.user_details_url(),
            params={
              'format': 'json',
              'oauth2_access_token': access_token
            }
        )
```

**Comment by JVUnderground** [comment]
Can we update the pip package, so we don't have to use unofficial installs?

**Comment by gennaroanesi** [comment]
getting this same issue here. got it working with @makslevental correction 👌

**Comment by makslevental** [comment]
any movement on this? this is a pretty big deal isn't it?

**Comment by Josue23** [comment]
Getting this same issue here.
Where do I put this code from @makslevental?

**Comment by makslevental** [comment]
@Josue23 https://github.com/python-social-auth/social-core/pull/179/files

**Comment by gennaroanesi** [comment]
any news on this one?

**Comment by drewjaja** [comment]
Currently using social core 1.5 to get around the issue.

**Comment by omab** [comment]
I've just released version `1.7.0` which carries the `Bearer` authentication method, that's the correct format documented by LinkedIn ([Step 4 — Make authenticated requests](https://developer.linkedin.com/docs/oauth2)). `oauth2_access_token` header is not needed.
